lib/modules/home/home_view.dart

```diff
@@ -139,4 +126,27 @@ class _HomeViewImplState
   void setCurrentPage(int page) {
     _currentPage.add(page);
   }
+
+  void _listenSliderMenuContainerAnimationController(Duration timestamp) {
+    final animationController =
+        _sliderMenuContainerKey.currentState?.animationController;
+    if (animationController == null) {
+      _widgetsBinding
+          .scheduleFrameCallback(_listenSliderMenuContainerAnimationController);
```

Comment:
why do you call this binding again inside `_widgetsBinding.scheduleFrameCallback` ? 🤔  Any reason why we need recursion please? 🙏

Reply:
@manh-t I try to check frame by frame until `SliderMenuContainer` is setup then `sliderMenuContainerKey.currentState?.animationController` will ready to use.

It seems it won't be setup at the first frame, so that I keep looking in the next and the next frame

Reply:
@markgravity okay, I think I have a trick here to help you fix this problem. You can use:
```dart
Future.delayed(Duration(milliseconds: 0), () {
      _listenSliderMenuContainerAnimationController();
});
```
Put this to `initState` and please let me know if it works 😄 
This is what I learned in [Flutter course](https://www.udemy.com/course-dashboard-redirect/?course_id=1708340), it helps us solve issue that BuildContext isn't ready to work when calling functions in `initState`

Reply:
also, don't forget to remove binding function when you try above function ☝️

Reply:
@manh-t let's me give it a shot 👍

Reply:
@manh-t It doesn't work, `sliderMenuContainerKey.currentState?.animationController` still nil when `Future.delayed` is called 😅

Reply:
![image](https://user-images.githubusercontent.com/60863885/123208236-e8502600-d4e8-11eb-8ac3-81b92f7333f8.png)
okay, I tried and Trung's approach worked, let's use this and remove recursion 😄

Reply:
@manh-t  The `build` only get called once and `schedulerPhase` returns `SchedulerPhase.persistentCallbacks`
So it can't get into this block 😁 
```dart
WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
        _listenSliderMenuContainerAnimationController();
});
```

Reply:
hmm, okay to speed up the process I will skip this issue temporarily 😂